dev/packages/beats/aws-0.0.1/docs/README.md

```diff
@@ -1 +1 @@
-TODO
\ No newline at end of file
+TODO
```

Comment:
Hmmm.. I'm afraid it might be a bit outdated. Could you run `mage ImportBeats` for the aws package? I think this file should be populated. If you face any problems, you can reach out to me directly!

Reply:
yes! done! Thank you for your help!